### ISSUE TITLE
ToChar().Single Issue

### ISSUE THREAD

**Issue by garno-fc** [awesome-inc/FontAwesome.Sharp#32]
I'm using markdown icons with your guys' library- first of all, great job! It's great to have something like this that supports WinForms. 

I'm running into an issue where getting a bitmap of the icon is throwing a 'Sequence contains multiple elements' exception. Since your project is open source, I looked into the method and troubleshooted (troubleshot? idk..) the issue down to the ToChar<TIcon>() method. It uses the .Single() instead of the .First() linq command (which is arguably more correct).

I'd love your guys' input!
Thanks!
Greg

**Comment by mkoertgen** [comment]
Hi @garno-fc, thanks for reaching out.

Did you narrow it down to this line?

- [IconHelper.cs#L47](https://github.com/awesome-inc/FontAwesome.Sharp/blob/master/FontAwesome.Sharp/WPF/IconHelper.cs#L47)

Yes, technically there could be returned a string with multiple chars, cf.:

- [char.ConvertFromUtf32](https://docs.microsoft.com/en-us/dotnet/api/system.char.convertfromutf32?view=netframework-4.8)

So, using `First()` instead of `Single()` would be the right way to go.

Since we have a unit test checking all possible fontawesome icons

- [IconHelper_Should.cs#L18](https://github.com/awesome-inc/FontAwesome.Sharp/blob/master/FontAwesome.Sharp.Tests/WPF/IconHelper_Should.cs#L18)

let me ask out of curiosity which utf8-codes did you use?

Would you mind adding/changing tests & preparing a PR?

**Comment by garno-fc** [comment]
Hello-
Yes, I narrowed it down to that line (because I was able to write that line in my code outside of fontawesome.sharp). I temporarily solved my problem by not drawing those icons in my icon chooser dialog.

I'm not sure which utf8-code I used, but I'm using basically your guys' MaterialDesignIcons test example and trying to display any of the icons in the screenshot. 

![image](https://user-images.githubusercontent.com/48067378/76231764-51743300-61eb-11ea-800a-abd6a2b721f3.png)

I looked up the one selected and it has a hex code in the enum file as 0xF0127, so 983335?? I'm really not sure how all of that works.

As far as adding/changing tests and preparing a pull request- I currently don't actually have FontAwesome.Sharp building/working on my machine (for whatever reason- I tried extensively yesterday). Additionally, I'm assuming you don't have unit tests working for third party icon fonts- so I don't know what I'd be able to contribute there.
